<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pdfreader</class>
 <widget class="QMainWindow" name="Pdfreader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPdfView" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QToolButton" name="layout">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/Resource/layout.png</normaloff>:/Resource/layout.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="zoomin">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/Resource/zoom-in.png</normaloff>:/Resource/zoom-in.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="zoomout">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/Resource/zoom-out.png</normaloff>:/Resource/zoom-out.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox"/>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QComboBox" name="zoomcomboBox">
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="actionSearchToggle">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/Resource/search.png</normaloff>:/Resource/search.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_5" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLineEdit" name="searchlineEdit">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="search">
            <property name="text">
             <string>搜索</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QSplitter" name="splitter">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <widget class="QTabWidget" name="tabWidget">
             <property name="maximumSize">
              <size>
               <width>250</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="currentIndex">
              <number>1</number>
             </property>
             <widget class="QWidget" name="tab">
              <attribute name="icon">
               <iconset resource="res.qrc">
                <normaloff>:/Resource/directory.png</normaloff>:/Resource/directory.png</iconset>
              </attribute>
              <attribute name="title">
               <string/>
              </attribute>
              <widget class="QTreeView" name="treeView">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>211</width>
                 <height>221</height>
                </rect>
               </property>
              </widget>
             </widget>
             <widget class="QWidget" name="tab_2">
              <attribute name="icon">
               <iconset resource="res.qrc">
                <normaloff>:/Resource/thumbnail.png</normaloff>:/Resource/thumbnail.png</iconset>
              </attribute>
              <attribute name="title">
               <string/>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QListView" name="thumbnailview"/>
               </item>
              </layout>
             </widget>
            </widget>
            <widget class="QWidget" name="widget_4" native="true"/>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>437</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>视图</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_3"/>
    <addaction name="action_4"/>
    <addaction name="action_5"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>         打开文件</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>放大</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>缩小</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>分页</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QPdfView</class>
   <extends>QWidget</extends>
   <header>qpdfview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
